demos/simple_pipeline/gpu-example-pipeline.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import kfp.dsl as kfp
+
+def training_op(learning_rate: float,
+                num_layers: int,
+                optimizer='ftrl',
+                step_name='training'):
+  return kfp.ContainerOp(
+    name=step_name,
+    image='katib/mxnet-mnist-example',
+    command=['python', '/mxnet/example/image-classification/train_mnist.py'],
```

Comment:
I don't want to invest more effort in this pipeline since it's not really what we want to be showing. I would rather use one of the [better examples](https://github.com/kubeflow/pipelines/tree/master/samples), but to do that we need katib support for tf-job, which @richardsliu is looking into. Pipeline DSL support for katib would round things out to turn this into a much smoother demo.